### PR TITLE
Fixed ticket (persist chat messages in cache):

### PR DESCRIPTION
**Pull request by fmunir1153**
This change is about persistence of a liquichain message (if user is offline), when he gets online and connect, it will get all persisted messages on UI and at that time all messages will be removed from cache.

https://github.com/telecelplay/mv-liquichain/issues/3

### CODE REVIEW

File: meveo-admin/ejbs/src/main/java/org/meveo/export/EntityExportImportService.java [hunk 1]
```diff
@@ -2189,6 +2193,7 @@ private void refreshCaches() {
         notificationCacheContainerProvider.refreshCache(null);
         customFieldsCacheContainerProvider.refreshCache(null);
         jobCacheContainerProvider.refreshCache(null);
+        liquichainCacheContainerProvider.refreshCache(null);
```
Comment (ClementBareth) [inline meveo-admin/ejbs/src/main/java/org/meveo/export/EntityExportImportService.java]: Maybe it's better not to mention liquichain here and rename this to userMessageCacheContainerProvider

File: meveo-admin/ejbs/src/main/java/org/meveo/service/technicalservice/wsendpoint/WebsocketServerEndpoint.java [hunk 2]
```diff
@@ -112,6 +115,19 @@ public void onOpen(Session session, EndpointConfig config, @PathParam("endpoint-
 			session.getUserProperties().put("endpointName", endpointName);
 			if (username != null) {
 				session.getUserProperties().put("username", username);
+
+				if(LIQUICHAIN_MODULE_CODE.equals(endpointName)){
```
Comment (ClementBareth) [inline meveo-admin/ejbs/src/main/java/org/meveo/service/technicalservice/wsendpoint/WebsocketServerEndpoint.java]: That's not good. You need to find a solution where the module code is not hardcoded

**Review by ClementBareth — CHANGES_REQUESTED** [review]

File: meveo-admin/ejbs/src/main/java/org/meveo/service/technicalservice/wsendpoint/WebsocketServerEndpoint.java (hunk 2 above)
Comment (fmunir1153) [reply meveo-admin/ejbs/src/main/java/org/meveo/service/technicalservice/wsendpoint/WebsocketServerEndpoint.java]: Resolved now

File: meveo-admin/ejbs/src/main/java/org/meveo/export/EntityExportImportService.java (hunk 1 above)
Comment (fmunir1153) [reply meveo-admin/ejbs/src/main/java/org/meveo/export/EntityExportImportService.java]: renamed

**Comment by fmunir1153** [comment]
Feedback Changes and other changes after testing committed and pushed to same branch.

**Review by ClementBareth — APPROVED** [review]

**Pull request merged**